src/runtime/applier/applier.cpp

```diff
@@ -120,12 +121,9 @@ namespace hpx { namespace applier
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    hpx::util::thread_specific_ptr<applier*, applier::tls_tag> applier::applier_;
-
     applier::applier(parcelset::parcelhandler &ph, threads::threadmanager& tm)
       : parcel_handler_(ph), thread_manager_(tm)
     {
-        init_tss();
```

Comment:
@hkaiser not fully understanding your changes in https://github.com/STEllAR-GROUP/hpx/pull/3599/commits/0e243f0d5f2811459f3f46cc5100d3d8adb70265, is this still going to to do the right thing with init_tss removed?

Reply:
Good question! I'll investigate.

Reply:
Ping @hkaiser?

Reply:
@msimberg this change is fine. Let's merge this PR.